src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist

```diff
@@ -311,11 +311,16 @@ method java.util.Map clear
 method java.util.Map containsKey java.lang.Object
 method java.util.Map entrySet
 method java.util.Map get java.lang.Object
+method java.util.Map getOrDefault java.lang.Object java.lang.Object
 method java.util.Map isEmpty
 method java.util.Map keySet
 method java.util.Map put java.lang.Object java.lang.Object
 method java.util.Map putAll java.util.Map
+method java.util.Map putIfAbsent java.lang.Object java.lang.Object
 method java.util.Map remove java.lang.Object
+method java.util.Map replace java.lang.Object java.lang.Object
+method java.util.Map replace java.lang.Object java.lang.Object java.lang.Object
+method java.util.Map replaceAll java.lang.Object
```

Comment:
I think you meant `java.util.function.BiFunction`? See test failure.

Reply:
Yes, that would've been better. I did not intend to add it, however - Java lambdas aren't nice single-function interfaces that play well with Groovy closures. I also avoided compute etc. which also require the lambda API.